src/components/InterimFindings/index.js

```diff
@@ -1,160 +1,128 @@
-          <p>1조</p>
-          <p>2조</p>
-          <p>3조</p>
-          <p>4조</p>
-        </div>
-        <div className="wrapRound">
-          <p>1R</p>
-          <div className="team one">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team two">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team three">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team four">
-            <span>X</span>
-            <span>100</span>
-          </div>
-        </div>
-        <div className="wrapRound">
-          <p>2R</p>
-          <div className="team one">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team two">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team three">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team four">
-            <span>X</span>
-            <span>100</span>
-          </div>
-        </div>
-        <div className="wrapRound">
-          <p>3R</p>
-          <div className="team one">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team two">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team three">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team four">
-            <span>X</span>
-            <span>100</span>
-          </div>
-        </div>
-        <div className="wrapRound">
-          <p>4R</p>
-          <div className="team one">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team two">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team three">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team four">
-            <span>X</span>
-            <span>100</span>
-          </div>
-        </div>
-        <div className="wrapRound">
-          <p>5R</p>
-          <div className="team one">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team two">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team three">
-            <span>X</span>
-            <span>100</span>
-          </div>
-          <div className="team four">
-            <span>X</span>
-            <span>100</span>
-          </div>
-        </div>
-      </WrapResult>
-    </ModalDiv>
+    <WrapResult>
+      <TeamNumber>
+        <p>1조</p>
+        <p>2조</p>
+        <p>3조</p>
+        <p>4조</p>
+      </TeamNumber>
+
+      {RoundData.map((it) => (
```

Comment:
it이 뭘까요? item일까요? 뭔가 props네임에 대해 고민을 좀 해보시면 좋을거 같습니다.
```suggestion
      {RoundData.map((item) => ( ||       {RoundData.map((table) => (
```

Reply:
iterator의 준말로 알고 있구 저도 이렇게 썼는데 좀더 찾아 보겠습니다!

Reply:
iterator 전혀 예상치 못한 줄임말이었네요!